src/redis-cli.c

```diff
@@ -6935,17 +6935,13 @@ static int clusterManagerCommandCreate(int argc, char **argv) {
     int replicas = config.cluster_manager_command.replicas;
     int masters_count = CLUSTER_MANAGER_MASTERS_COUNT(node_len, replicas);
     if (masters_count < 3) {
-        clusterManagerLogErr(
-            "*** ERROR: Invalid configuration for cluster creation.\n"
-            "*** Redis Cluster requires at least 3 master nodes.\n"
-            "*** This is not possible with %d nodes and %d replicas per node.",
-            node_len, replicas);
```

Comment:
I think would be useful to show the calculated number of masters and replicas, before showing the *"Redis Cluster requires at least 3 master nodes (...) are you sure?"* prompt.

How about this?

```Suggestion
        clusterManagerLogInfo("Requested to create a cluster with %d masters "
                              "and %d replicas per master.", node_len, replicas);
```